docs/griptape-cloud/knowledge-bases/accessing-data.md

```diff
@@ -0,0 +1,33 @@
+# Accessing Data in a Knowledge Base
+
+You can `Search` or `Query` the Knowledge Base for information contained in your Data Sources. `Search` will return a natural language response while `Query` will return the individual entries. Use whichever one best fits your use case.
+
+## From the Cloud Console
+
+You can try out both actions from the cloud console on the `Test` tab of your Knowledge Base.
+
+## From the API
+
+You can enact both `Query` and `Search` via the API by hitting their respective endpoints using a [Griptape Cloud API Key](https://cloud.griptape.ai/configuration/api-keys) and the Knowledge Base ID found on the `Config` tab of your Knowledge Base. 
+
+The following example commands will send the query string "test query string" and return the results from the Knowledge Base.
+
+### Query
+
+```shell
+export GT_CLOUD_API_KEY=<your API key here>
+export KNOWLEDGE_BASE_ID=<your knowledge base id here>
+curl -H "Authorization: Bearer ${GT_CLOUD_API_KEY}" --json '{"query": "test query string"}' https://cloud.griptape.ai/api/knowledge-bases/${KNOWLEDGE_BASE_ID}/query
```

Comment:
can we give clearer example to differentiate Query from Search in our example here? `test query string` is very vague.

Reply:
They are intended to be the same inputs as we want to show that both take a natural language input.